app/views/school/internet/mobile/extra_data_requests/guidance.html.erb

```diff
@@ -0,0 +1,55 @@
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs(items: [
+      { t('page_titles.school.internet.home.show.title') => internet_school_path(@school) },
+      t('page_titles.request_extra_mobile_data')
+    ],
+    user: @user,
+    school: @school) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.request_extra_mobile_data') %>
+    </h1>
+
+    <h2 class="govuk-heading-m">1. Check who is eligible for extra mobile data</h2>
+
+    <p class="govuk-body">You can request extra mobile data for children who:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>have no broadband connection at home</li>
+      <li>cannot afford additional data</li>
+    </ul>
+
+    <p class="govuk-body">You can make up to 10 requests for <%= govuk_link_to 'specific circumstances', specific_circumstances_school_path %>.</p>
+
+    <p class="govuk-body">More requests will be available when you report a closure or that 15 children are self-isolating.</p>
```

Comment:
We're not doing the limit yet, so we need to change this slightly to:

```One of the following must also apply. They are children:

* in years 3 to 11 and whose face-to-face education is disrupted
* who are extremely clinically vulnerable and need to shield on current official advice (this could be from a doctor or hospital consultant)
* who live in a household that’s been advised to shield because a family member is clinically extremely vulnerable
* who cannot attend school – even though theirs is open – because they live in a different area with local coronavirus restrictions```

Reply:
@fofr Just to confirm: does this replace the two paragraphs from 'You can make...' to '...15 children are self-isolating.' ?

Reply:
@fofr Thanks. I can see the changes on the prototype.